presto-native-execution/presto_cpp/main/tests/QueryContextManagerTest.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/QueryContextManager.h"
+#include <gtest/gtest.h>
+
+namespace facebook::presto {
+
+class QueryContextManagerTest : public testing::Test {};
+
+TEST_F(QueryContextManagerTest, removePrefix) {
+  EXPECT_EQ(
```

Comment:
We should test findOrCreateQueryCtx API to see if the session properties have been handled properly.